README.rst

```diff
@@ -23,3 +23,13 @@ a `-`), the default tagged image will be `ORG/IMG:DIST-vX-test` (instead of
 
 `make push` will push all the images that would be generated by `make build`
 (same rules apply for `TAG`).
+
+Internal `cachalot` user
+========================
+
+As part of base image generation, user named `cachalot` will be created. It is
+intended to be used to run tests without accidental root access which may affect
+the result.
+
+This user is added to sudoers with a permission to run any sudo command without
+a password.
```

Comment:
Sorry, I don't know how I didn't catch this earlier, but I think the main intention of this user is to not have to build a new image just to install dependencies, and being able to use `sudo apt install` as part of the build process instead.  Normally test were ran as users anyway. Maybe that should be mentioned in the readme and release notes.

Reply:
Updated. Like this?